api/server.go

```diff
@@ -24,13 +26,16 @@ type Server struct {
 
 	apiKey string
 	store  Store
+
+	log *logger.Logger
 }
 
 // New creates a new API server.
-func New(apiKey string, store Store) *Server {
+func New(apiKey string, store Store, obsrv *observe.Observer) *Server {
 	s := &Server{
 		apiKey: apiKey,
 		store:  store,
+		log:    obsrv.Log,
```

Comment:
```suggestion
		log:    obsrv.Log.With(lctx.Str("component", "api")),
```

This makes it so that you can see which components logs come from :) Do this whenever you pass a logger to a `New`